src/SDDPoptimize.jl

```diff
@@ -52,8 +52,8 @@ function solve_SDDP(model::SPModel,
     end
 
     # Run SDDP upon example:
-    run_SDDP!(model, param, V, problems, display)
-    return V, problems
+    ic = run_SDDP!(model, param, V, problems, display)
+    return V, problems, ic
 end
```

Comment:
Ok because I can duplicate run_SDDP! and SDDP_solve using the surchage but don't know if it is really usefull. Let's speak about that on the next meeting